doc/Overview/Logging/LoggingOverview.md

```diff
@@ -1,108 +1,174 @@
+                    )
+        });
+...
 ```
 
 ## Serilog
 
-```csharp
-private IHost Host { get; }
+Serilog is a third-party logging framework that is otherwise available as a [NuGet package](https://www.nuget.org/packages/Serilog). The benefit of using Serilog is that it provides a more robust logging framework that can be configured to log to a variety of different sinks, including the console and file. Examples of other sinks include Azure Application Insights, Seq, and many more.
+
+An option to use Serilog is available by calling the `UseSerilog()` extension method on the `IHostBuilder` instance. This method will configure the Serilog logger to specifically not use the console or file sink by default. The following example shows how to enable Serilog:
 
-public App()
+```csharp
+protected override void OnLaunched(LaunchActivatedEventArgs e)
 {
-    Host = UnoHost
-        .CreateDefaultBuilder()
-        .UseLogging()
-        .UseSerilog()
-        .Build();
-    // ........ //
-}
+    var appBuilder = this.CreateBuilder(args)
+        .Configure(host => {
+            host
+            .UseLogging()
+            .UseSerilog();
+        });
+...
 ```
 
-## Logging
+For more information about Serilog, check out [Getting started with Serilog](https://github.com/serilog/serilog/wiki/Getting-Started).
 
-To log, you simply need to get a `ILogger` and use the appropriate methods.
+## Uno Internal Logging
+
+To use the same logging system for Uno internal messages call `UseLogging()` on the `IHost` instance:
 
 ```csharp
-var myLogger = myServiceProvider.GetService<ILogger<MyType>>();
+private IHost Host { get; }
 
-myLogger.LogInformation("This is an information log.");
+protected override void OnLaunched(LaunchActivatedEventArgs e)
+{
+    var appBuilder = this.CreateBuilder(args)
+        .Configure(host => {
+            host
+            .UseLogging()
+        });
+        
+    Host = appBuilder.Build();
+...
 ```
 
-Alternatively you can add `ILogger` as a constructor parameter for any type that's being created by the dependency injection container.
-
+### Setting the XAML Log Level
 
-## Uno Internal Logging
+Sometimes it's necessary to filter messages recorded for specific XAML types to reduce noise. When a preference is specified for the **XAML log level**, the logger will change the verbosity of events from a set of XAML-related types.
 
-To use the same logging for Uno internal messages call EnableUnoLogging after the Build method on the IHost instance. 
+It can be set by calling the `XamlLogLevel()` extension method on the `ILoggingBuilder` instance. The following example shows how to set the XAML log level to `Information`:
 
 ```csharp
-private IHost Host { get; }
+protected override void OnLaunched(LaunchActivatedEventArgs e)
+{
+    var appBuilder = this.CreateBuilder(args)
+        .Configure(host => {
+            host.UseLogging(
+                builder => {
+                    builder.XamlLogLevel(LogLevel.Information);
```

Comment:
I'd suggest listing the various extension methods here, along with the set of namespaces they adjust the filter for